journalpump/journalpump.py

```diff
@@ -24,7 +23,7 @@
 import time
 import uuid
 
-GeoIPReader: Union[Type[GeoIPProtocol], None]
+GeoIPReader: Any
```

Comment:
Isn't it worse going to a less concrete type?

Reply:
Ah, this is a leftover. So originally we didn't have _any_ types. Adding a protocol for the bits of GeoIP we used was my initial solution, but since I don't have a solid grasp on what the behaviour should be it puts too tight of a bound on what is known - so I pulled it out and just set GeoIPReader to `Any` because in practice that's kind of how it's handled by the code.

Reply:
Oh this wasn't a leftover - there was going to be a much more complicated protocol type, but it's a lot of work for an ill-defined typespace so it was dropped in favor of Any - the code dynamically tests for features anyway.